src/Node/Expression/GetAttrExpression.php

```diff
@@ -36,7 +36,7 @@ public function __construct(AbstractExpression $node, AbstractExpression $attrib
             trigger_deprecation('twig/twig', '3.15', \sprintf('Not passing a "%s" instance as the "arguments" argument of the "%s" constructor is deprecated ("%s" given).', ArrayExpression::class, static::class, $arguments::class));
         }
 
-        parent::__construct($nodes, ['type' => $type, 'ignore_strict_check' => false, 'optimizable' => true], $lineno);
+        parent::__construct($nodes, ['type' => $type, 'ignore_strict_check' => false, 'optimizable' => true, 'is_optional_chain' => $isOptionalChain], $lineno);
```

Comment:
Do we need another attribute for the optional chain ? Couldn't it be a matter of the `ignore_strict_check` attribute ?